lte/gateway/c/core/oai/test/spgw_task/test_spgw_procedures.cpp

```diff
@@ -620,5 +629,527 @@ TEST_F(SPGWAppProcedureTest, TestReleaseBearerError) {
   std::this_thread::sleep_for(std::chrono::milliseconds(END_OF_TEST_SLEEP_MS));
 }
 
+MATCHER_P2(check_params_in_actv_bearer_req, lbi, tft, "") {
+  auto cb_req_rcvd_at_mme =
+      static_cast<itti_s11_nw_init_actv_bearer_request_t>(arg);
+  if (cb_req_rcvd_at_mme.lbi != lbi) {
+    return false;
+  }
+  if (!(cb_req_rcvd_at_mme.s1_u_sgw_fteid.teid)) {
+    return false;
+  }
+  if ((memcmp(
+          &cb_req_rcvd_at_mme.tft, &tft, sizeof(traffic_flow_template_t)))) {
+    return false;
+  }
+  return true;
+}
```

Comment:
move to the top before the class definition.

Reply:
Fixed in latest commit https://github.com/magma/magma/pull/10521/commits/f8cb61a2dbbb95ffbe3a44f6d14e8e88c5f520fd